ndmg/utils/gen_utils.py

```diff
@@ -353,14 +153,54 @@ def get_dir_info(self):
         return scans
 
 
-def all_strings(iterable_):
+def make_initial_directories(outdir: Path, parcellations=[]) -> None:
     """
```

Comment:
Maybe have this function return a list or dict, and then could keep filepaths in one place